cmake_format/configuration.py

```diff
@@ -31,6 +31,10 @@ def parse_bool(string):
   return False
 
 
+def parse_as_list(string, sepr=":"):
+  return list(string.split(sepr))
```

Comment:
@dkandyba What means parameter's name "sepr"? I think will be better "separator", if I have understood right.